testing/test_preamble.sh

```diff
@@ -133,3 +133,27 @@ function build_if_not_release {
     fi
     cmd/build $build_mode missing
 }
+
+function activate_venv {
+    if [ -d venv ]; then
+        echo Activating venv
+        source venv/bin/activate
+    fi
+
+    if [ -n "$DAQ_CODECOV" ]; then
+        echo Running Python with codecov analysis...
+        PYTHON_CMD="coverage run -a"
+    else
+        PYTHON_CMD="python3"
+    fi
+
+    export PYTHON_CMD
+}
+
+function deactivate_venv {
```

Comment:
this is odd... why do you need to do this?  Either the system should be running inside of venv or not (raw install)... why do you need to flip in and out?

Reply:
> Either the system should be running inside of venv or not
Right, makes sense. I thought only the varz_state_collector should be run in venv... Removed this function. PTAL..